modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -535,17 +590,15 @@ private void createFile() {
                     log.error("Error while creating the output directory.");
                 }
             }
-
             sequenceNumber = getLogFileSequenceNumber();
             outputFileTimeStamp = System.currentTimeMillis();
             fstream = new OutputStreamWriter(new FileOutputStream(new File(logDir + "/output-" +
-                                                                                    siddhiAppContextName + "-" +
-                                                                                    sequenceNumber + "-" +
-
-                                                                                   (outputFileTimeStamp)
-                                                                                   + ".csv")
-                                                                        .getAbsoluteFile()), StandardCharsets
-                                                     .UTF_8);
+                                                                            siddhiAppContextName + "-" +
+                                                                            sequenceNumber + "-" +
+                                                                            ".csv")
```

Comment:
Why the id not added?

Reply:
https://github.com/wso2/product-sp/pull/855/commits/3971112ad2ea08bef38437ea24bb12cad020692c